specification/xpack/usage/types.ts

```diff
@@ -0,0 +1,429 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Phases } from '@ilm/_types/Phases'
+import { Dictionary } from '@spec_utils/Dictionary'
+import { ByteSize, EmptyObject, Field, Name } from '@_types/common'
+import { Job, JobStatistics } from '@_types/ml/Job'
+import { integer, long, uint, ulong } from '@_types/Numeric'
+import { Statistics } from '@_types/slm/Statistics'
+
+export class Usage {
```

Comment:
Let's rename this type to `Base`, so the type name will be `XPack.Usage.Base` :)

```suggestion
export class Base {
```

Reply:
done